cli/lib/exec/run.js

```diff
@@ -74,6 +74,10 @@ const processRunOptions = (options = {}) => {
     args.push('--headed', options.headed)
   }
 
+  if (options.noExit) {
+    args.push('--no-exit')
```

Comment:
There are no tests for this, and the logic is broken because you're not testing `options.exit === false` in here